.github/workflows/buf-ci.yaml

```diff
@@ -0,0 +1,20 @@
+name: Buf CI
+on:
+  push:
+    paths:
+      - 'proto/**'
+permissions:
+  contents: read
+jobs:
+  buf:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: bufbuild/buf-action@v0.1
+        with:
+          username: ${{ secrets.BUF_USER }}
+          token: ${{ secrets.BUF_TOKEN }}
+          version: 1.32.2
```

Comment:
Current is [1.31](https://github.com/bufbuild/modules/blob/22fffa6bf7b9667f1de8af7c756bc43133cf7adb/Makefile#L10) maybe update all to be consistent? 1.34.0 is the latest.

Reply:
updated